src/game/messages/client.rs

```diff
@@ -284,4 +284,7 @@ pub enum ClientMessage {
     CastSkillSelf(SkillSlot),
     CastSkillTargetEntity(SkillSlot, ClientEntityId),
     CastSkillTargetPosition(SkillSlot, Point2<f32>),
+    RunToggle(),
```

Comment:
Do not need () for enum variants with no data, I would have expected rust fmt to remove them.

e.g. this can just be:
```
RunToggle,
SitToggle,
DriveToggle,
```